packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -163,6 +163,19 @@ Media.createMediaConnection = (
   // we might not have any TURN server if TURN discovery failed or wasn't done or
   // we might get an empty TURN url if we land on a video mesh node
   if (turnServerInfo?.url) {
+    if (!isBrowser('firefox')) {
```

Comment:
Limiting it to non-firefox, as we might be not handling it properly with Full-ICE. Once we test it properly with Firefox, we should enable it also for Firefox.